tools/apiview/parsers/rust-api-parser/CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 # 1.1.0 
 
-Enhanced rendering of `use` items to more closely match the rustdoc (docs.rs) HTML view:
-- Improved handling of `use` items and module re-exports with better tracking to prevent duplication
-- Refactored module processing with enhanced sorting and organization of child items
-- Fixed bugs and improved utility functions for better code maintainability
+1. Enhanced rendering of `use` items to more closely match the rustdoc (docs.rs) HTML view:
+    - Improved handling of `use` items and module re-exports with better tracking to prevent duplication
+    - Refactored module processing with enhanced sorting and organization of child items
+    - Fixed bugs and improved utility functions for better code maintainability
+2. Updated `ReviewLine#LineId`s to stable, meaningful identifiers rather than relying on the dynamic rustdoc ids.
```

Comment:
Don't use numbered lists for CHANGELOGs. We never do that.

Reply:
cool

Reply:
```suggestion
- Enhanced rendering of `use` items to more closely match the rustdoc (docs.rs) HTML view:
    - Improved handling of `use` items and module re-exports with better tracking to prevent duplication
    - Refactored module processing with enhanced sorting and organization of child items
    - Fixed bugs and improved utility functions for better code maintainability
- Updated `ReviewLine#LineId`s to stable, meaningful identifiers rather than relying on the dynamic rustdoc ids.
```